src/BloomBrowserUI/bookEdit/readerSetup/readerSetup.ui.ts

```diff
@@ -490,6 +490,7 @@ function handleThingsToRemember(jqueryEvent: JQueryEventObject): void {
  */
 function updateSightWords(ta: HTMLInputElement): void {
   var words: string = ta.value.trim().replace(/ ( )+/g, ' '); // remove consecutive spaces
+  words = words.replace(/,/g, ''); // remove commas
```

Comment:
There are actually only 2 places, it just looks like 4 because we are checking in the generated JavaScript files along with the source TypeScript files. I will see if it is feasible to create a function that is shared between the 2 files.